### PR TITLE
Fix Param(s) + Fix dupli Gang / House + Last BIS_fnc_param -> Param(s) + Fix fn_wantedPerson.sqf

### PR DESCRIPTION
**Pull request by NiiRoZz**
#892 
#894

### CODE REVIEW

File: life_client/core/housing/fn_sellHouse.sqf [hunk 1]
```diff
@@ -8,6 +8,8 @@
     Sells the house and delete all container near house.
 */
 private["_house","_uid","_action","_houseCfg"];
+if((time - life_action_delay) < 2) exitWith {hint localize "STR_NOTF_ActionDelay"};
```
Comment (Jawshy) [inline life_client/core/housing/fn_sellHouse.sqf]: > [ERROR: Possible missing white space after if detected at ../Framework/life_client/core/housing/fn_sellHouse.sqf Line number: 11](https://travis-ci.org/ArmaLife/Framework/builds/148016155)

Add a space after the `if`.

File: life_client/dialog/function/fn_gangWithdraw.sqf [hunk 2]
```diff
@@ -7,17 +7,20 @@
     Withdraws money from the gang bank.
 */
 private["_value"];
+if((time - life_action_delay) < 2) exitWith {hint localize "STR_NOTF_ActionDelay"};
```
Comment (Jawshy) [inline life_client/dialog/function/fn_gangWithdraw.sqf]: > [ERROR: Possible missing white space after if detected at ../Framework/life_client/dialog/function/fn_gangWithdraw.sqf Line number: 10](https://travis-ci.org/ArmaLife/Framework/builds/148016155)

Add a space after the `if`.

File: life_client/core/housing/fn_sellHouse.sqf (hunk 1 above)
Comment (NiiRoZz) [inline life_client/core/housing/fn_sellHouse.sqf]: Thanks @jawshy

**Comment by BoGuu** [comment]
What @tkcjesse pointed out in your last params pull request

**Comment by tkcjesse** [comment]
> What @tkcjesse pointed out in your last params pull request

Meaning, why not cleanup the private arrays...?

**Comment by BoGuu** [comment]
@danielstuart14 

You not around on Gitter atm?

**Comment by tkcjesse** [comment]
I mean... he's not around discord ever. Why would he be there?

**Pull request merged**